src/routes/RemediationDetails.js

```diff
@@ -169,6 +209,9 @@ class RemediationDetails extends Component {
                                                                 { pluralize(totalSystems, 'system') }
                                                             </DescriptionList>
                                                         </FlexItem>
+                                                        { playbookRuns &&
+                                                            this.renderLatestActivity(playbookRuns)
```

Comment:
Can we make this so that once the user submits the Execute button modal this part of the page auto-refreshes so that the Latest activity (which is going to be the one the user just submitted) shows up here without the user having to refresh the page?

Reply:
Also, when the latest activity is in the running state can we re-fetch the list of playbook runs (e.g. every 30 seconds) for the Latest activity status to auto-update?